.ci/ci_check_commit.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+# "Copyright [2019]"
+# @ function: Scan code with cobra 
+# @ require : Make sure your machine is linux (centos/ubuntu), yum or apt is ready
+# @ author  : asherli
+# @ file    : ci_check_commit.sh
+# @ date    : 2019
+
+set -e
+
+scan_code_script="python ~/cobra/cobra.py -t "
+
+LOG_ERROR() {
+    content=${1}
+    echo -e "\033[31m${content}\033[0m"
+}
+
+LOG_INFO() {
+    content=${1}
+    echo -e "\033[32m${content}\033[0m"
+}
+
+execute_cmd() {
+    command="${1}"
+    eval ${command}
+    ret=$?
+    if [ $ret -ne 0 ];then
+        LOG_ERROR "FAILED of command: ${command}"
+        exit 1
+    else
+        LOG_INFO "SUCCESS of command: ${command}"
+    fi
+}
+
+init()
```

Comment:
this is unnecessary, what about just use HEAD^ below

Reply:
It happened when repo only has first commit, it's better than nothing